lib/path.js

```diff
@@ -1342,11 +1342,10 @@ const posix = {
   },
 
 
-  _makeLong: function _makeLong(path) {
+  toLongUNCPath: function toLongUNCPath(path) {
```

Comment:
This is a new API, it need to be consistent.
Since the windows implementation uses `resolve` either the posix one does too, or remove it from the Windows one.